File: src/main/java/com/test/guess/Game.java
package com.test.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secretNum = random.nextInt(10) + 1;
        System.out.println(secretNum);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secretNum) {
            System.out.println("Please enter a number");
            number = scanner.nextInt();
            if (number > secretNum) {
                System.out.println("Lower!");
            } else if(number < secretNum) {
                System.out.println("Higher!");
            } else {
                System.out.println("Correct!");
            }
        }

/*        int i = 0;
        do {
            System.out.println(i);
            i++;
        } while (i < 3); {

        }*/
    }
}

File: src/main/kotlin/com/test/Game.kt
package com.test

import java.util.*

fun main() {
    var secretNum = Random().nextInt(10) + 1;
    println(secretNum);
//    val scanner = Scanner(System.`in`);
    var number = 0
    while (number != secretNum) {
        println("Please enter a number")
//        number = scanner.nextInt()
        number = readLine()!!.toInt();
        if (number > secretNum) {
            println("Lower!")
        } else if (number < secretNum) {
            println("Higher!")
        } else {
            println("Correct!")
        }
    }
}